src/Symfony/Component/Config/Tests/Definition/Dumper/YamlReferenceDumperTest.php

```diff
@@ -32,7 +31,7 @@ private function getConfigurationAsString()
 acme_root:
     boolean:              true
     scalar_empty:         ~
-    scalar_null:          ~
+    scalar_null:          null
```

Comment:
why is it dumped as ``null`` here while all other places using null are dumping it as ``~`` ?